src/jdk.incubator.vector/share/classes/jdk/incubator/vector/ByteVector.java

```diff
@@ -3920,6 +3920,7 @@ void intoMemorySegment0Template(Class<M> maskClass, MemorySegment ms, long offse
 
     // End of low-level memory operations.
 
+    @ForceInline
```

Comment:
Why this change? Was it missing before or you found that based on testing?
What is criteria to add `@ForceInline`?

Reply:
okay.